src/App.js

```diff
@@ -17,18 +17,22 @@ class App extends Component {
       isLoading: true,
       filteredMovies: [],
       beingSearched: false,
-      error: ''
+      error: '',
+      searchInput: ''
```

Comment:
Good job making this it's own state! Its understodd that now we're going to be clearing this based off some event that will change the state